src/lib/pickles/pickles.ml

```diff
@@ -194,6 +194,14 @@ module Make_str (_ : Wire_types.Concrete) = struct
         local_max_proofs_verifieds
         H1.T(Proof_.Messages_for_next_proof_over_same_field.Wrap).t ) =
     let dummy_chals = Dummy.Ipa.Wrap.challenges in
+    let rev_magic :
```

Comment:
We should fix the iterator below and remove this. Best not to lie to the typechecker!